output/json.go

```diff
@@ -24,48 +25,58 @@ type JSONReport struct {
 	Behaviors map[string]*JSONBehaviorReport `json:"behaviors"`
 }
 
-var JSON ReporterFunc = func(config *plan.Config, tests <-chan execute.TestResponse) Summary {
-	summary := Summary{}
-	report := JSONReport{
-		Behaviors: make(map[string]*JSONBehaviorReport),
+type JSON struct {
+	r    JSONReport
+	path string
+}
+
+func (j *JSON) Start(config *plan.Config) error {
+	j.path = config.JSONReportPath
+	j.r.Behaviors = make(map[string]*JSONBehaviorReport)
+
+	if config.JSONReportPath == "" {
+		return fmt.Errorf("JSON_REPORT_PATH is a required environment varialbe for REPORT=json")
```

Comment:
For now, I'd say lets leave it until we allow multiple config sources, eg env and yaml file